build/docker/Dockerfile.compiler

```
@@ -4,12 +4,26 @@ ARG OS=ubuntu
 FROM falkordb/falkordb-build:$OS AS builder
 
+ARG OS
+
 ARG DEBUG=0
 
 WORKDIR /FalkorDB
 
 COPY . /FalkorDB
 
+# if $OS is rhel8 install gcc 12
+RUN if [ "$OS" = "rhel8" ]; then \
```

Hadolint (2.14.0)

[error] Expected 'fi'. Fix any mentioned problems and try again.

(SC1072)